spec/spec_helper.rb

```diff
@@ -75,3 +91,19 @@ def in_tmp_dir
     end
   end
 end
+
+private
+
+def stringify_for_fastlane(hash)
+  hash.map do |key, value|
+    # rubocop:disable Style/CaseLikeIf
+    if value.is_a?(Hash)
+      "#{key}: {\n#{stringify_for_fastlane(value)}\n}"
+    elsif value.is_a?(String)
+      "#{key}: \"#{value}\""
```

Comment:
This will cause trouble if:
(1) `value` contains some interpolation or backslash-escapes, as they would then be interpreted
(2) `value` contains literal quotes in it (e.g. if `value = 'translators: "%1$@" is the app name'` — which could totally be a value you want to pass to your future replacement of the action generating the `.po` file from scratch if you wanted to pass the comments to write when generating each block…)

I'm wondering if just using `.inspect` instead of implementing your own `stringify_for_fastlane` wouldn't be enough?
Sure it would not format it as nicely (e.g. it would generate `{:paths=>[\"foo\", \"bar\\\"x\\\".txt\"], :destination=>\"out.txt\"}` instead of a nice `{ paths: ["foo", "bar\"x\".txt"], destionation: "out.txt" }` with nice newlines and all… but I don't think we care, as even if as humans we tend to use the latter syntax (implicit Hash and `key: value` instead of `:key => value` syntax, to make it look more like if it were named parameters) both syntaxes are valid and I think the former should still be recognized by `Fastfile.new.parse(…)` anyway, which is the important part…